starters/academic/content/post/getting-started/index.md

```diff
@@ -47,15 +47,15 @@ categories:
 2. The template can be modified and customised to suit your needs. It's a good platform for anyone looking to take control of their data and online identity whilst having the convenience to start off with a **no-code solution (write in Markdown and customize with YAML parameters)** and having **flexibility to later add even deeper personalization with HTML and CSS**
 3. You can work with all your favourite tools and apps with hundreds of plugins and integrations to speed up your workflows, interact with your readers, and much more
 
-{{< figure src="https://raw.githubusercontent.com/wowchemy/wowchemy-hugo-modules/master/academic.png" title="The template is mobile first with a responsive design to ensure that your site looks stunning on every device." >}}
+[![The template is mobile first with a responsive design to ensure that your site looks stunning on every device.](/.github/media/academic.png)](https://wowchemy.com)
```

Comment:
> /.github/media/academic.png

Hugo does not publish the .github folder, so the change to the image path that you suggested will not work.